tests/sqlparser_clickhouse.rs

```diff
@@ -220,6 +220,88 @@ fn parse_create_table() {
     );
 }
 
+#[test]
+fn parse_create_table_with_primary_key() {
+    match clickhouse().one_statement_parses_to(
+        concat!(
+            r#"CREATE TABLE db.table (`i` Int, `k` Int)"#,
+            " ENGINE=SharedMergeTree('/clickhouse/tables/{uuid}/{shard}', '{replica}')",
+            " PRIMARY KEY tuple(i)",
+            " ORDER BY tuple(i)",
```

Comment:
We can probably add a test case that the parser rejects e.g. `ORDER BY tuple(i), tuple(k)`

Reply:
Added verification of this error into this unit test.